src/types.mligo

```diff
@@ -242,8 +235,6 @@ type forbid_xtz_params =
   | Drop_proposal of proposal_key
```

Comment:
As I was suggesting elsewhere, I think that `migratable_parameter` (and its comment) should no longer exist, and instead be "merged" into the `parameter` itself.

So IMO parameter should look like:
```ocaml
type parameter =
  (allow_xtz_params, "", forbid_xtz_params, "") michelson_or
```
where `migratable_parameter` no longer exists and `forbid_xtz_params` stays the same, but `allow_xtz_params` also contains `transfer_contract_tokens_param`:
```ocaml
type allow_xtz_params =
  | CallCustom of custom_ep_param
  | Propose of propose_params
  | Transfer_contract_tokens of transfer_contract_tokens_param
```
(because it allows xtz transfers).